clients/drcachesim/common/utils.h

```diff
@@ -59,6 +59,10 @@ namespace drmemtrace {
 #define INVALID_THREAD_ID 0
 // We avoid collisions with DR's INVALID_PROCESS_ID by using our own name.
 #define INVALID_PID -1
+// A separate sentinel for an idle core with no software thread.
+// XXX i#6703: Export this in scheduler.h as part of its API when we have
+// the scheduler insert synthetic headers.
+#define IDLE_THREAD_ID -1
```

Comment:
Perhaps this should be a different constant than INVALID_PID?

Reply:
It is a different namespace (different record).  There's not another great choice: -2?  Leaning toward leaving it.